vendor/onethinx/otx18.yaml

```diff
@@ -0,0 +1,116 @@
+name: Onethinx LoRaWAN module
+description: Onethinx dual core module with integrated LoRa and LoRaWAN stack, SX126x radio chips, integrated secure element and an integrated omnidirectional antenna.
+
+# Hardware versions (optional, use when you have revisions)
+hardwareVersions:
+  - version: '18.02'
+    numeric: 1
+
+# Firmware versions (at least one is mandatory)
+firmwareVersions:
+  - # Firmware version
+    version: '1.0'
+    numeric: 1
```

Comment:
please add the corresponding hardware version for this firmware in this case '18.02'

Reply:
I am not sure what to add under firmware version, or how to form it. Currently out latest version is 0x000000C1 (0xC1 for short, 193 in decimal), but we plan to release C2 in the comming days (194 decimal). All firmware versions that we release work on this single hardware version that we offer. difference is in minor bug fixes. Version naming is in hexadecimal.

Older clients use older versions. Should I name all of them? All version are built based on 1.0.2B.

Reply:
Disregard, I now see I need to add hardwareVersion to firmwareVersions